components/cam/src/physics/cam/shoc.F90

```diff
@@ -1714,7 +1717,7 @@ subroutine fterms_input_for_diag_third_shoc_moment(&
   thedz2 = 1._rtype/(dz_zt+dz_zt_kc)
 
   iso       = isotropy_zi
-  isosqrd   = iso**2
+  isosqrd   = bfb_square(iso)
```

Comment:
This change isn't BFB, is it?

Reply:
In the case that you're changing the Fortran to be BFB-equal to C++, this creates a non-BFB PR that should arguably be separate from the C++ conversion work.

Reply:
Or maybe I'm misunderstanding, and all these changes went through in a prior PR? If so, please ignore this comment.

Reply:
These changes are included in https://github.com/E3SM-Project/scream/pull/599. Once it is merged and I rebase to master these will not be here. Should I change to [WIP] until that PR is merged?

Reply:
No, that's fine. I'm just juggling enough things to be distracted, is all. :-) Sorry for the noise.

Reply:
No worries at all! I have so many interdependent PRs going on.. ha